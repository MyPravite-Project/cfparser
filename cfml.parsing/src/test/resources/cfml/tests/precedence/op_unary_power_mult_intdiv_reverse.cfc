7\1*+x^4;