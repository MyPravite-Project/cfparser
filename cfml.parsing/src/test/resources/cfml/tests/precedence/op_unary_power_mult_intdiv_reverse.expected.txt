/*===TOKENS===*/
INTEGER_LITERAL          <7>
'\'                      <\>
INTEGER_LITERAL          <1>
'*'                      <*>
'+'                      <+>
IDENTIFIER               <x>
'^'                      <^>
INTEGER_LITERAL          <4>
';'                      <;>
/*===TREE===*/
(scriptBlock
  (element
    (statement
      (startExpression
        (baseExpression
          (baseExpression (unaryExpression (primaryExpression (literalExpression 7))))
          \
          (baseExpression
            (baseExpression (unaryExpression (primaryExpression (literalExpression 1))))
            *
            (baseExpression
              (baseExpression
                +
                (baseExpression (unaryExpression (memberExpression (identifier x))))
              )
              ^
              (baseExpression (unaryExpression (primaryExpression (literalExpression 4))))
            )
          )
        )
      )
      ;
    )
  )
)
/*======*/
/*===DECOMPILE===*/
7 \ 1 * +x ^ 4
/*======*/